src/main/java/nl/tudelft/ti2206/group9/ShaftEscape.java

```diff
@@ -80,6 +77,7 @@ public static void showPopup(final Popup popup) {
 
 	/** Exits the Application. */
 	public static void exit() {
+		SaveGameWriter.saveGame("firstActualGame.json");
```

Comment:
This will make the Writer only save the game when the "exit" button is pressed in the main menu. When the red X of the window is pressed, nothing happens.

<logic,review